avm/res/network/p2s-vpn-gateway/tests/e2e/waf-aligned/main.test.bicep

```
@@ -15,7 +15,7 @@ param resourceGroupName string = 'dep-${namePrefix}-network.p2svpngateway-${serv
 param resourceLocation string = deployment().location
 
 @description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
-param serviceShort string = 'vscwaf'
+param serviceShort string = 'npvgmin'
```

```suggestion
param serviceShort string = 'npvgwaf'
```

`min` only applies to the defaults test